.scripts/dfx-identity.sh

```diff
@@ -11,7 +12,15 @@ ALICE_PEM="$ALICE_HOME/.config/dfx/identity/default/identity.pem"
 BOB_PEM="$BOB_HOME/.config/dfx/identity/default/identity.pem"
 DEFAULT_PEM="$HOME/.config/dfx/identity/default/identity.pem"
```

Comment:
This should be 
`DEFAULT_PEM="$HOME/.config/dfx/identity/$(dfx identity whoami)/identity.pem"`
to allow any dfx identity to be used

Reply:
@ozwaldorf that can be quite useful, sounds very good! So, that'll be out of the scope for this PR, do you mind providing a PR on that, please? It'll require making other changes, which includes docs.